Gemfile.lock

```diff
@@ -373,3 +380,9 @@ DEPENDENCIES
   tzinfo-data
   web-console
   webdrivers
+
+RUBY VERSION
+   ruby 3.1.0p0
+
+BUNDLED WITH
+   2.3.10
```

Comment:
Should be removed

Reply:
We need to run bundle for minimagick, and this gets printed.